custom-layer-example/webpack.config.js

```diff
@@ -0,0 +1,22 @@
+module.exports = {
+  output: {
+    libraryTarget: "umd",
+    filename: "bundle.js",
+    library: "CustomLayerLibrary"
+  },
+  entry: {
+    main: "./index.js"
+  },
+  externals: {
+    "@deck.gl/layers": {
+      commonjs: "deck",
```

Comment:
- OCD: Same order of externals for layers and core? 
- Worth adding comments about why `root`/`commonjs`/`commonjs2` are needed, or that is too much clutter? 

Maybe just a link to the webpack doc page that explains this? I have not used these before, so I don't know what they do...